Project.toml

```
@@ -1,7 +1,7 @@
 name = "Manifolds"
 uuid = "1cead3c2-87b3-11e9-0ccd-23c62b72b94e"
 authors = ["Seth Axen <seth.axen@gmail.com>", "Mateusz Baran <mateuszbaran89@gmail.com>", "Ronny Bergmann <manopt@ronnybergmann.net>", "Antoine Levitt <antoine.levitt@gmail.com>"]
-version = "0.6.7"
+version = "0.6.8"
```

```suggestion
version = "0.6.9"
```
(the one we merge today will have 8)

---

I think we'll have to make a small merge commit anyway :wink:

---

Sure, I just noticed, when I checked whether I have to bump further over on the other PR.